src/components/Post.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useState } from 'react';
+
+const Post = () => {
+    
+    const [likeCount, setLikeCount] = useState(0);
+    const updateLikes = () => {
+        setLikeCount(likeCount+1);
+    };
```

Comment:
This Post component is not being used anywhere, but this code here would be great o have in the App component!